source/module_cell/module_symmetry/symmetry.cpp

```diff
@@ -1939,11 +1939,12 @@ void Symmetry::hermite_normal_form(const ModuleBase::Matrix3 &s3, ModuleBase::Ma
     ModuleBase::TITLE("Symmetry","hermite_normal_form");
     // check the non-singularity and integer elements of s
     assert(!equal(s3.Det(), 0.0));
-    auto round = [](double x){return (x>0.0)?floor(x+0.5):ceil(x-0.5);};
-    ModuleBase::matrix s=s3.to_matrix();
+    auto round = [](double x) {return (x > 0.0) ? floor(x + 0.5) : ceil(x - 0.5);};
```

Comment:
STL lib has round function?

Reply:
OK, I've changed it to `std::round`. (but not stl)